immich/docker-compose.yml

```diff
@@ -0,0 +1,129 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: immich-proxy
+      APP_PORT: 8080
+      PROXY_AUTH_ADD: "false"
```

Comment:
Is this turned off to allow third party apps to talk to immich's APi?

Reply:
Yes, Immich does have mobile apps for iOS and Android. Immich does also require a login. So I came to the conclusion that an additional auth middleware wasnt needed. Is this correct?

Thank you for your review and feedback!!